File: app/src/main/java/com/rachelleignacio/listbucket/domain/interactors/DeleteListInteractor.java
package com.rachelleignacio.listbucket.domain.interactors;

import com.rachelleignacio.listbucket.domain.interactors.base.Interactor;

/**
 * Created by rachelleignacio on 3/10/17.
 */

public interface DeleteListInteractor extends Interactor {
    interface Callback {
        void onListDeleted(int adapterPosition);
    }
}

File: app/src/main/kotlin/com/rachelleignacio/listbucket/domain/interactors/DeleteListInteractor.kt
package com.rachelleignacio.listbucket.domain.interactors

import com.rachelleignacio.listbucket.domain.interactors.base.Interactor

/**
 * Created by rachelleignacio on 8/9/17.
 */
interface DeleteListInteractor : Interactor {
    interface Callback {
        fun onListDeleted(adapterPosition: Int)
    }
}